src/Tribe/Cost_Utils.php

```diff
@@ -290,8 +290,8 @@ public function parse_cost_range( $costs, $max_decimals = null ) {
 			if ( preg_match_all( '/' . $price_regex . '/', $cost, $matches ) ) {
 				$cost = reset( $matches );
 			} else {
-				$matches = array();
-				$cost = array( 0 );
+				$cost = array();
```

Comment:
Rather than defaulting strings to `0`, instead default them to a non-cost (an empty array).